script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_9.txt

```diff
@@ -174,7 +174,7 @@ The headache caused by my exposed eyes blends with my own rage.
 I don't believe this. Of all the things that could happen, that bastard managed to see through me.
 It must have been his intuition.
 An awareness of death that surpasses that of a beast.
-Despite having no reason to think so, that guy had a sudden intuition that the enemy had a means to kill him, and was afraid. Since he recognized me as a threat, he leapt away without even a second's hesitation.//Ruby emphasis 敵には自分を殺す手段があると直感し
+Despite having no reason to think so, that guy had a sudden intuition that the enemy had a means to kill him, and was afraid. Since he recognized me as a threat, he leaped away without even a second's hesitation.//Ruby emphasis 敵には自分を殺す手段があると直感し,  I think leapt is UK english
```

Comment:
I'm an american and i think "leaped" sounds worse than leapt.

Reply:
Should we add this as an exception in the rules/linter?

Reply:
I don't like it either

Reply:
![image](https://user-images.githubusercontent.com/93603030/147312144-b9025c9a-12aa-45b7-b6d3-49932092f5bd.png)